Sources/Subprocess/Teardown.swift

```diff
@@ -88,7 +85,7 @@ extension Execution {
         using sequence: some Sequence<TeardownStep> & Sendable,
         on processIdentifier: ProcessIdentifier
     ) async {
-        await withUncancelledTask {
+        await withUncanceledTask {
```

Comment:
This was actually the preferred spelling with two `l`s.

Reply:
Interestingly Xcode thinks `Uncancelled` is misspelled